zerver/finders.py

```diff
@@ -11,7 +11,7 @@ class ExcludeUnminifiedMixin(object):
     def list(self, ignore_patterns):
         # We can't use ignore_patterns because the patterns are
         # applied to just the file part, not the entire path
-        excluded = '^(js|styles|templates)/'
+        excluded = '^(js|templates)/'
```

Comment:
Without this tweak, collectstatic gave file-not-found sort of errors only when the newer version of pipeline was used. I tried a lot to find out why this was happening but I couldn't. I thought there's no harm in unminified CSS being shipped out, so I finally made this tweak to make things work.